mender/values.yaml

```
@@ -31,8 +31,8 @@ global:
     CONTAINER_NAME: mender-artifact-storage
   smtp:
     EMAIL_SENDER: root@localhost
-    SMTP_ADDRESS: "localhost:25"
-    SMTP_LOGIN: "null"
+    SMTP_HOST: "localhost:25"
+    SMTP_USERNAME: "null"
     SMTP_PASSWORD: "null"
     SMTP_SSL: "false"
```

Without `SMTP_AUTH_MECHANISM` I'm getting template errors:
```suggestion
    SMTP_SSL: "false"
    SMTP_AUTH_MECHANISM: "PLAIN"
```